### ISSUE TITLE
Strange all-static-files behavior

### ISSUE THREAD

**Issue by patkar** [allinurl/goaccess#818]
If I use `--all-static-files` the bandwidth and total request files goes up and on static files down. But I can see how a `.css?queryparam` is moved out of request files. I think there is something strange :smirk: 

| | Bandwidth | Total |
| - | - | - |
| Requested files, called without | 300.28 GiB | 658.835 |
| Static files, called without | 174.66 GiB | 34.795 |
| Request files, w/ `--all-static-files` | 390.04 GiB | 677.300 |
| Static files, w/ `--all-static-files` | 84.9 GiB | 16.330 |

Latest 1.2, built on Ubuntu 14.04

**Comment by allinurl** [comment]
Not sure I follow what's the issue here. Shouldn't 658.835 + 34.795 = 677.300 + 16.330? Thanks

**Comment by patkar** [comment]
Yep, the total of requests in both reports is the same. But when I call `goaccess --all-static-files` I see how requests like `/dist/main.css?q=232534243` get moved to the "Static files" panel, but the total count (and bandwidth) of "Static files" goes down. I expect it to go up, when it counts more static files with `--all-static-files`

**Comment by allinurl** [comment]
Thanks for the reply. I'm trying to reproduce this on my side, however, my numbers do go up. Are you able to replicate this with a smaller data set? Thanks
